akka-distributed-data/src/main/scala/akka/cluster/ddata/LWWMap.scala

```diff
@@ -31,6 +31,13 @@ object LWWMap {
  * use a timestamp value based on something else, for example an increasing version number
  * from a database record that is used for optimistic concurrency control.
  *
+ * The `defaultClock` is using max value of `System.currentTimeMillis()` and `currentTimestamp + 1`.
```

Comment:
Not really about this PR but, isn't that unsafe since currentTimeMillis isn't monotonic?